pkg/prowloader/prow.go

```diff
@@ -201,6 +203,39 @@ func (pl *ProwLoader) prowJobToJobRun(pj prow.ProwJob) error {
 	return nil
 }
 
+func (pl *ProwLoader) findOrAddPullRequests(refs *prow.Refs) []models.ProwPullRequest {
+	if refs == nil {
+		return nil
+	}
+	pulls := make([]models.ProwPullRequest, 0)
+
+	for _, pr := range refs.Pulls {
+		if pr.Link == "" {
+			continue
+		}
+
+		pull := models.ProwPullRequest{}
+		pl.dbc.DB.Where("link = ?", pr.Link).Find(&pull)
+		pull.Org = refs.Org
+		pull.Repo = refs.Repo
+
+		pull.Link = pr.Link
+		pull.SHA = pr.SHA
```

Comment:
We can have multiple SHA's tested, so I'm wondering if it shouldn't be unique on SHA instead of link. Or SHA + Link.

Reply:
Is SHA always going to be the last one we saw? Or do you want one entry per SHA being tested?

Do we want to track SHA at all?  I'd assume we're mostly just interested in linking to the PR itself.

Reply:
The way it's written now, yea we'll have the last SHA we saw.  

If we track it by SHA (i.e. possibly multiple entries per PR), we can differentiate between: "it took a lot of testing to get lucky on this sha" vs. "developer kept pushing changes, that's why this pr had so many test runs."